migrations/update_user_notifications.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ *
+ * phpBB Browser Push Notifications. An extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2024, phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\webpushnotifications\migrations;
+
+use phpbb\db\migration\migration;
+
+class update_user_notifications extends migration
+{
+	/**
+	 * @inheritDoc
+	 */
+	public static function depends_on()
+	{
+		return ['\phpbb\webpushnotifications\migrations\add_webpush'];
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function effectively_installed()
+	{
+		$sql = 'SELECT method
+			FROM ' . $this->table_prefix . "user_notifications
+			WHERE method = '" . $this->db->sql_escape('notification.method.phpbb.wpn.webpush') . "'";
```

Comment:
Maybe add `AND notify = 1` as far as we're looking for enabled notification types.

Reply:
For this I'm just looking to see if it exists yet, because if it does it means it's already been installed and people have started making settings already.